packages/openneuro-app/src/scripts/datalad/dashboard/datasets/dataset-tab.jsx

```diff
@@ -120,7 +115,7 @@ const DatasetTab = ({
     ) : (
       <ErrorBoundary error={error} subject={'error in dashboard dataset tab'}>
         <DatasetTabLoaded
-          data={data}
+          datasets={data.datasets || data.search}
```

Comment:
I think this fork is not the best way to handle this but it keeps this reusable for now.